src/components/Modals/Send/views/Address.tsx

```diff
@@ -37,13 +37,11 @@ export const Address = () => {
     formState: { errors },
   } = useFormContext<SendInput>()
   const address = useWatch<SendInput, SendFormFields.Address>({ name: SendFormFields.Address })
+  const input = useWatch<SendInput, SendFormFields.Input>({ name: SendFormFields.Input })
   const { send } = useModal()
   const asset = useWatch<SendInput, SendFormFields.Asset>({ name: SendFormFields.Asset })
   const isYatFeatureEnabled = useFeatureFlag('Yat')
 
-  // reset address when asset changes
```

Comment:
this was correctly clearing the field when the asset changed, but also causing the scanned QR code to erroneously clear